src/ServiceManagement/StorSimple/Commands.StorSimple/Cmdlets/GetAzureStorSimpleResourceContext.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Microsoft.WindowsAzure.Commands.StorSimple.Exceptions;
+using Microsoft.WindowsAzure.Commands.StorSimple.Properties;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.WindowsAzure.Commands.StorSimple.Cmdlets
+{
+    /// <summary>
+    /// This commandlet will return the currently selected resource. If no resource is selected will throw a ResourceContextNotFoundException
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "AzureStorSimpleResourceContext"),OutputType(typeof(StorSimpleResourceContext))]
+    public class GetAzureStorSimpleResourceContext : StorSimpleCmdletBase
+    {
+        protected override void BeginProcessing()
+        {
+            //we expliclity override BeginProcessing() so that it doesnt verify resource selection as part of StorSimpleCmdletBase
+            //class's BeginProcessing method
```

Comment:
Thanks for the comment which describes such weird code. In this case, what about have a protected boolean flag in the base class which is set to false when creating `GetAzureStorSimpleResourceContext` so it does not verify? This is one option getting rid of an empty overrided method